src/model/company.profile.ts

```diff
@@ -0,0 +1,19 @@
+export interface PTFCompanyProfile {
+    companyName: string;
+    companyNumber: string;
+    companyStatus: string;
+    companyType: string;
+    address: {
+        line_1: string;
+        line_2: string;
+        postCode: string;
+    };
+    accountsDue: string;
+    accountingPeriodStartOn: string;
+    accountingPeriodEndOn: string;
+    isAccountsOverdue: boolean;
+    isConfirmationStatementOverdue: boolean;
+    confirmationStatementDue: string;
+    incorporationDate: string;
+    ptfRequested: string;
```

Comment:
Wondering if we need this now? What is it for?

Reply:
These will be required as we will need to display information about confirmation stements alongside accounts - there work to do to get this data from the sdk.